src/event_store_ms/src/main/java/config/Config.java

```diff
@@ -0,0 +1,36 @@
+package config;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.ConnectionFactory;
+import java.sql.Connection;
+import java.io.IOException;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class Config {
+
+    public static void setConfigFactory(ConnectionFactory factory) {
+        factory.setHost("localhost");
+        factory.setVirtualHost("/victor_travels");
+        factory.setUsername("admin");
+        factory.setPassword("admin");
+    }
+
+    public static void setConfigQueue(Channel channel, String queueName) throws IOException {
+        channel.queueDeclare(queueName, true, false, false, null);
```

Comment:
Wg mnie tutaj queueDeclare jest niepotrzebne bo kolejka już istnieje w rabbitcie. Wystarczy się do niej tylko podłączyć. Czy to jest na pewno wymagane?

Reply:
done